### ISSUE TITLE
Crash if AllowDragHandler is assigned to a <Draggable> 

### ISSUE THREAD

**Issue by AndreZinner** [Postlagerkarte/blazor-dragdrop#17]
When I assign a handler for AllowDrag in my component with

```
<Draggable Tag="@SomeItem" AllowDrag="(d) => AllowDrag(d)">

<code>
    public bool AllowDrag(DraggableItem item)
    {
       return true;
    }
</code>

```

the OnDragStart() method in Dropzone.razor crashes with the following error message:

> Microsoft.CSharp.RuntimeBinder.RuntimeBinderException: Delegate 'System.Func<Blazor.DragDrop.Core.DraggableItem,bool>' has some invalid arguments
>    at CallSite.Target(Closure , CallSite , Func`2 , Object )
>    at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
>    at Blazor.DragDrop.Core.Dropzone.OnDragStart(Int32 draggableId)
>    at Blazor.DragDrop.Core.Dropzone.<>c__DisplayClass0_0.<BuildRenderTree>b__1()
>    at Microsoft.AspNetCore.Components.EventCallbackWorkItem.InvokeAsync[T](MulticastDelegate delegate, T arg)
>    at Microsoft.AspNetCore.Components.EventCallbackWorkItem.InvokeAsync(Object arg)
>    at Microsoft.AspNetCore.Components.ComponentBase.Microsoft.AspNetCore.Components.IHandleEvent.HandleEventAsync(EventCallbackWorkItem callback, Object arg)
>    at Microsoft.AspNetCore.Components.EventCallback.InvokeAsync(Object arg)
>    at Microsoft.AspNetCore.Components.RenderTree.Renderer.DispatchEventAsync(UInt64 eventHandlerId, EventFieldInfo fieldInfo, EventArgs eventArgs)
> 

```
...
 private void OnDragStart(int draggableId)
    {

        var dragedItem = DragDropService.GetDraggablesForDropzone(_dropzoneId).Single(x=> x.Id == draggableId);

        if (dragedItem.AllowDrag != null && !dragedItem.AllowDrag(dragedItem.Tag)) return;

...

```

I think, the "dragItem.Tag" should be replaced by a "dragItem", because the handler expects a DraggableItem (and not the "dynamic item" stored in Tag)

```
...
 private void OnDragStart(int draggableId)
    {

        var dragedItem = DragDropService.GetDraggablesForDropzone(_dropzoneId).Single(x=> x.Id == draggableId);

        if (dragedItem.AllowDrag != null && !dragedItem.AllowDrag(dragedItem)) return;

...

```

**Comment by Postlagerkarte** [comment]
Hi @AndreZinner - thank you for reporting this!  

If you create a pull request I will merge it.

**Comment by AndreZinner** [comment]
Sorry, i have no git installed and I am not able to create a pull request.

**Comment by Postlagerkarte** [comment]
No worries - I will fix it.

**Comment by Postlagerkarte** [comment]
@AndreZinner Fixed in v1.1.6
